jmh-core/src/main/java/org/openjdk/jmh/profile/XCTraceAsmProfiler.java

```diff
@@ -0,0 +1,276 @@
+/*
+ * Copyright (c) 2024, Oracle and/or its affiliates. All rights reserved.
```

Comment:
I was not sure about the copyright header, so plunked the Oracle's one.

Reply:
Yeah, that's fine.